### ISSUE TITLE
Error during Pod Install

### ISSUE THREAD

**Issue by KishPatel1996** [ProjectSeptemberInc/gl-react-native#94]
Getting the error below 

```
    - ERROR | attributes: Missing required attribute `authors`.
    - WARN  | attributes: Missing required attribute `license`.
    - ERROR | attributes: Missing required attribute `summary`.
    - WARN  | license: Missing license type.
    - WARN  | source: Git sources should specify a tag.

```

In my podfile, I added the line 

```
pod 'RNGL', :path => '../node_modules/gl-react-native'
```

Any idea of what's causing this, or how to go around it?

**Comment by gre** [comment]
@mrspeaker : do you know about this?

**Comment by mrspeaker** [comment]
Cocoapods 3 requires authors and summary meta info (for some stupid reason). [PR](https://github.com/ProjectSeptemberInc/gl-react-native/pull/95) just adds these fields (and updates the versions)

**Comment by gre** [comment]
fixed in v2.28.3
